CHANGES.md

```diff
@@ -3,6 +3,9 @@
 
 - Fix versioning of artifact variables (eg %{cmxa:...}), which were introduced
   in 2.0, not 1.11. (#3149, @nojb)
+  
+- Fix a bug introduced in 2.3.0 where dune insists on using [fswatch] on linux 
+  (even when [inotifywait] is available). (#3162, @aalekseyev)
```

Comment:
Nit: the correct way to quote in markdown is `fswatch` and `inotifywait`.

Reply:
Thanks, I forgot I wasn't writing OCaml there.